cypress/e2e/room_board/files_in_board_card/addDeleteImageFileInRoomBoard.feature

```diff
@@ -0,0 +1,75 @@
+@regression_test
+@stable_test
+Feature: Room - Upload, download and delete image file type in the Room Board
+
+    As a teacher, I want to upload, download and delete image file in the room board so that I can easily share and manage the board contents.
+
+    Scenario: Upload, download and delete image file in the room board, including pre & post conditions
+
+        # pre-condition: creating accounts
+        Given I am logged in as a '<teacher>' at '<namespace>'
+
+        # pre-condition: room, board and card are existing
+        Given I am logged in as a '<teacher>' at '<namespace>'
+        Given a room named '<room_name>' exists
+        Given a multi-column board named '<board_title>' exists in the room
+        Given the multi-column board has a column with a card
+
+        # teacher uploads image file in the multi-column room board
+        When I click on the three dot on the card
+        When I click on the option edit in the three dot menu on the card
+        When I click on plus icon to add content into card
+        Then I see the dialog box to select element for the card
+        When I click on the button Close on the element selection dialogue box
```

Comment:
typo `dialogue` and in other places as well

Reply:
done!